src/index.ts

```diff
@@ -117,6 +137,7 @@ export class CdktfProviderProject extends cdk.JsiiProject {
         name: "team-tf-cdk",
         email: "github-team-tf-cdk@hashicorp.com",
       },
+      majorVersion: getMajorVersion(options.outdir),
```

Comment:
Maybe add something like this?
```suggestion
      // sets major version to 1 for the first version but resets it for future versions to allow them to automatically increase to e.g. v2 if breaking changes occured
      majorVersion: getMajorVersion(options.outdir),
```